src/khoj/processor/conversation/utils.py

```diff
@@ -306,3 +315,22 @@ def reciprocal_conversation_to_chatml(message_pair):
 def remove_json_codeblock(response: str):
     """Remove any markdown json codeblock formatting if present. Useful for non schema enforceable models"""
     return response.removeprefix("```json").removesuffix("```")
+
+
+def get_image_from_url(image_url: str, type="pil"):
```

Comment:
Can we use some basic typing here?

```
class ImageWithType:
    encoded_data: Any
    type: str
```

Otherwise code like `get_image_from_url(item["image_url"]["url"])[0]` gets pretty confusing to follow. Would become `get_image_from_url(item["image_url"]["url"]).encoded_data` with proposed changes.

Reply:
Would need to update references in `src/khoj/processor/conversation/google/utils.py` and `src/khoj/processor/conversation/anthropic/utils.py`.

Reply:
Fixed